File: plugin_fly_news/src/main/java/com/fly/tour/news/MainNewsActivity.java
package com.fly.tour.news;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.fly.tour.common.base.BaseActivity;
import com.fly.tour.news.fragment.MainNewsFragment;

public class MainNewsActivity extends BaseActivity {

    @Override
    public int onBindLayout() {
        return R.layout.activity_news_main;
    }

    @Override
    public void initView() {
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_content,MainNewsFragment.newInstance()).commit();
    }

    @Override
    public void initData() {

    }
}

File: plugin_fly_news/src/main/java/com/fly/tour/news/MainNewsActivity.kt
package com.fly.tour.news

import com.fly.tour.common.base.BaseActivity
import com.fly.tour.news.fragment.MainNewsFragment

class MainNewsActivity : BaseActivity() {
    override fun onBindLayout(): Int {
        return R.layout.activity_news_main
    }

    override fun initData() {
        supportFragmentManager.beginTransaction().replace(R.id.frame_content, MainNewsFragment.Companion.newInstance()).commit()
    }
}